admin/src/templates/logout_confirm.html

```diff
@@ -0,0 +1,11 @@
+<mm-modal
+  status="status"
+  danger="true"
+  title-key="'Confirm'"
+  submit-key="'yes'"
+  cancel-key="'no'"
+  on-cancel="cancel()"
+  on-submit="submit()"
+>
+  <p translate>confirm.logout</p>
+</mm-modal>
```

Comment:
I don't think you need this template any more, right?

Reply:
We don't,I've removed it in the next commit.